src/dataflow/src/render/join/delta_join.rs

```diff
@@ -543,18 +543,22 @@ where
 
     let mut datums = DatumVec::new();
     let mut row_builder = Row::default();
-    let (oks, errs2) = dogsdogsdogs::operators::half_join(
+    let (oks, errs2) = dogsdogsdogs::operators::half_join::half_join_internal_unsafe(
         &updates,
         trace,
         |time| time.saturating_sub(1),
         comparison,
+        |timer, _count| timer.elapsed().ge(&std::time::Duration::from_millis(10)),
         // TODO(mcsherry): consider `RefOrMut` in `half_join` interface to allow re-use.
```

Comment:
nit: We could move this comment to an issue in TimelyDataflow/differential-dataflow.